config.ru

```diff
@@ -1,3 +1,12 @@
 # frozen_string_literal: true
+require_relative 'lib/cleanup/cleanup.rb'
+
+at_exit do
+  cleanup = Cleanup.new
+  cleanup.shutdown
```

Comment:
Oh it throws exceptions if something goes wrong, took me forever to get it right :-)